### DISCUSSION TITLE
Generic type of table/collection

### DISCUSSION THREAD

**Discussion by florianbepunkt** [prisma/prisma#11250] [Q&A]
I'm trying to migrate from IndexedDB to Prisma with SQLite. I have some functional repository setup that is shared by all entities.

What I'm looking for is a generic type that describes the db actions that Prisma allows. I'm not looking for middleware as it does not fit the.

```ts
const create = (tableOrCollection: WhichTypeIsThis<A>) => (entity: A) => {
  // do some stuf
  
  // call the prisma method
  tableOrCollection.create({data: entity})

  // do some more stuff
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
